lib/search_reference_title_normaliser.rb

```diff
@@ -0,0 +1,5 @@
+class SearchReferenceTitleNormaliser
```

Comment:
Namespacing might help here? `SearchReferences::TitleNormaliser` ?

Reply:
NB: You probably already know but make sure to not use `SearchReference` (singular) since that will play up with the autoloader and the existing model

Reply:
Sounds good to me :)